06_modules_lab/01.py

```diff
@@ -2,3 +2,9 @@
 from sys.argv import and prints "Hello Python"
 count times
 """
+import sys
+
+if len(sys.argv)==2:
+    times = sys.argv[1]
+    for i in range(0,int(times)):
```

Comment:
Better to convert to int a line before (when reading from sys.argv[1] to times) and here use: `for i in range(times):`